<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIEV - Sistema Integrado de Evaluación Vestibular</string>
  </property>
  <property name="objectName">
   <string>main-window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="objectName">
    <string>central-widget</string>
   </property>
   <layout class="QHBoxLayout" name="main_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <!-- Activity Bar (VS Code style) - Always visible -->
     <widget class="QWidget" name="activity_bar">
      <property name="objectName">
       <string>activity-bar</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>48</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>48</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="activity_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <!-- App Logo -->
        <widget class="QPushButton" name="app_logo_btn">
         <property name="text">
          <string>S</string>
         </property>
         <property name="objectName">
          <string>app-logo</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>SIEV Medical System</string>
         </property>
        </widget>
       </item>
       <item>
        <!-- Separator -->
        <widget class="QFrame" name="separator_1">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>1</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="top_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>12</height>
          </size>
         </property>
        </spacer>
       </item>
       <!-- Activity Icons -->
       <item>
        <widget class="QPushButton" name="btn_dashboard">
         <property name="text">
          <string>▊</string>
         </property>
         <property name="objectName">
          <string>btn_dashboard</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Dashboard</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_patients">
         <property name="text">
          <string>⚤</string>
         </property>
         <property name="objectName">
          <string>btn_patients</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Pacientes</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_vng">
         <property name="text">
          <string>◉</string>
         </property>
         <property name="objectName">
          <string>btn_vng</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Evaluación VNG</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reports">
         <property name="text">
          <string>📄</string>
         </property>
         <property name="objectName">
          <string>btn_reports</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Reportes</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="middle_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <!-- Bottom Icons -->
       <item>
        <widget class="QPushButton" name="btn_settings">
         <property name="text">
          <string>⚙</string>
         </property>
         <property name="objectName">
          <string>btn_settings</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Configuración</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <!-- Collapsible Sidebar -->
     <widget class="QWidget" name="sidebar">
      <property name="objectName">
       <string>sidebar</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="sidebar_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <!-- Sidebar Header -->
        <widget class="QWidget" name="sidebar_header">
         <property name="objectName">
          <string>sidebar-header</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="sidebar_header_layout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="sidebar_title">
            <property name="text">
             <string>EXPLORER</string>
            </property>
            <property name="objectName">
             <string>sidebar-title</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="sidebar_header_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_collapse_sidebar">
            <property name="text">
             <string>‹</string>
            </property>
            <property name="objectName">
             <string>btn-collapse</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Collapse Sidebar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <!-- Sidebar Content Area -->
        <widget class="QWidget" name="sidebar_content">
         <property name="objectName">
          <string>sidebar-content</string>
         </property>
         <layout class="QVBoxLayout" name="sidebar_content_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <!-- Dynamic content will be loaded here based on active tab -->
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <!-- Main Editor Area -->
     <widget class="QWidget" name="content_area">
      <property name="objectName">
       <string>content-area</string>
      </property>
      <layout class="QVBoxLayout" name="content_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <!-- Tab Bar -->
        <widget class="QWidget" name="header">
         <property name="objectName">
          <string>header</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="header_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <!-- Active Tab -->
           <widget class="QLabel" name="page_title">
            <property name="text">
             <string>Dashboard</string>
            </property>
            <property name="objectName">
             <string>page-title</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>35</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="header_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <!-- Action buttons in header -->
          <item>
           <widget class="QPushButton" name="btn_theme_toggle">
            <property name="text">
             <string>☀</string>
            </property>
            <property name="objectName">
             <string>btn_theme_toggle</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Toggle Theme</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_notifications">
            <property name="text">
             <string>🔔</string>
            </property>
            <property name="objectName">
             <string>btn_notifications</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Notifications</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_user_menu">
            <property name="text">
             <string>👤</string>
            </property>
            <property name="objectName">
             <string>btn_user_menu</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="toolTip">
             <string>User Menu</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="header_right_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <!-- Central Plugin Area -->
        <widget class="QWidget" name="central_area">
         <property name="objectName">
          <string>central-area</string>
         </property>
         <layout class="QVBoxLayout" name="central_layout">
          <property name="objectName">
           <string>plugin_container</string>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <!-- Plugin content loads here -->
         </layout>
        </widget>
       </item>
       <item>
        <!-- Status Bar -->
        <widget class="QWidget" name="status_bar">
         <property name="objectName">
          <string>status-bar</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="status_layout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="status_info">
            <property name="text">
             <string>Ready</string>
            </property>
            <property name="objectName">
             <string>status-info</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="status_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="status_connection">
            <property name="text">
             <string>● Connected</string>
            </property>
            <property name="objectName">
             <string>status-connection</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
